### PR TITLE
Show controller model in model dropdown list

### LINKED ISSUE #3572: controller model shows in my profile model list but not the drop down for model picker
dropdown
profile-list

### PR DESCRIPTION
**Pull request by hatched**
Fixes #3572

### CODE REVIEW

**Comment by hatched** [comment]
#### To QA

Bootstrap your own controller and then use guiproxy to point this branch's GUI to it.
The model dropdown list should have the controller in it.

Opening the profile and you should not see a trash icon for the controller model in the list.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8572/

File: jujugui/static/gui/src/app/components/profile/model-list/test-model-list.js [hunk 1]
```diff
@@ -699,6 +699,149 @@ describe('Profile Model List', function() {
     expect(output).toEqualJSX(expected);
   });
 
+  it('does not show the trash icon for controller models', () => {
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/profile/model-list/test-model-list.js]: Should we implement this kind of checks with enzyme now?

**Comment by frankban** [comment]
QA ok.

File: jujugui/static/gui/src/app/components/profile/model-list/test-model-list.js (hunk 1 above)
Comment (hatched) [reply jujugui/static/gui/src/app/components/profile/model-list/test-model-list.js]: yes but I didn't want to do the enzyme conversion in this branch. So I figured we'd convert it all in one swoop.

**Comment by hatched** [comment]
Thanks! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**